Update README.md (#16)
l3lackShark/gosumemory: README.md

# gosumemory

Yet another memory reader for osu! Supports both Linux and Windows. (**requires sudo on Linux** though since only root can read /proc)

Build custom pp counters with ease!\
You can contact us here: https://discord.gg/8enr4qD

# Real-World examples:

> Created by [cyperdark](https://osu.ppy.sh/users/9893708):\

Reviewer: This list should be a representation of capabilities of this program, not to show yourself off, I believe you already have enough shout-outs, remove this
```suggestion
```

[![Alumetri](https://img.shields.io/badge/Alumetri%20v1.2---?style=for-the-badge&color=FF94B6)](https://mega.nz/file/QV1gTKoI#j1QRjDkrjnFvIhyb9JuGi3g_0XZCFzXEXz9PKWcxgmI)
[![Sotarks](https://img.shields.io/badge/Sotarks%20v1.0---?style=for-the-badge&color=C63F55)](https://mega.nz/file/oAlmlQoY#8ABeJPGboMLgCiaY5vR21HX2Km--_jiwqRHOmUJvVmg)
[![Mrekk](https://img.shields.io/badge/Mrekk%20v1.0---?style=for-the-badge&color=72a0d4)](https://mega.nz/file/UZsEUKDK#Ji3JAUr8_04Q7u0RG1BAJFGzZ2-CRhRZkEQqdXVrv60)
[![Mathi](https://img.shields.io/badge/Mathi%20v1.5---?style=for-the-badge&color=4981CE)](https://mega.nz/file/5dsk1QJD#noUKykU5qJYv53I2DPZ7PY2CIQOftS1ufqzOh4rqOb8)
[![RyuK](https://img.shields.io/badge/RyuK%20v1.0---?style=for-the-badge&color=f72f4d)](https://mega.nz/file/dY8k1YyZ#1Phdta1CzxXDotjtllUKsZunnCdliYlQ1VrZ_BNaNIs)

# Usage
     
1. [Download the latest Release](https://github.com/l3lackShark/gosumemory/releases/latest)
    * Unzip files anywhere
    > In the root folder of the program, you can find the **static** directory. It contains all of the available counters. Those are getting streamed via HTTP-File server

2. Run gosumemory & osu!

> 3. Run obs.exe

Reviewer: Why are we bringing extensions here? Either use them everywhere or nowhere.

>     * Open this link: http://localhost:24050 and choose counter which you liked.

Reviewer: ```suggestion
    * Visit this link in your browser: http://localhost:24050 and choose the counter that you like.
```

>     * Add new browser with that link, and set size ([all sizes of each counter](https://github.com/l3lackShark/static))

Reviewer: ```suggestion
    * Add a browser source in OBS (Width and Height could be found in the **Included counters** spoiler)
```
